src/scss/components/_footer.scss

```diff
@@ -0,0 +1,19 @@
+//------------------------------------------------------------------------------
+// @component: Footer
+//------------------------------------------------------------------------------
+// @description
+//
+//------------------------------------------------------------------------------
+
+//
+// @settings
+
+// Layout Variables
+
+
+//
+// @scss
+.c-footer {
+	text-align: center;
+	border-top: groove;
```

Comment:
remove this it looks really bad. also the text-align should really be applied to the individual elements and not the block for future reference but no big deal here